docs/help/publish-documentation/release-management.md

```diff
@@ -0,0 +1,70 @@
+# Release management
+
+Any new deployment made from the [web console](/getting-started/quick-start.md#step-2-upload-your-file), or from the [CLI](/bump-cli.md#deploy-a-file), or from the [Github Action](/continuous-integration/github-actions.md#deploy-on-git-push), will be visible in your deployments history list from your documentation dashboard, by selecting the Deployments tab.
+
+<div style={{textAlign: 'center'}}>
+
+![](/files/help/deployments-list.png)
+
+</div>
+
+On this list your can see the state of each deployment (`enqueued`, `deploying`, `deployed` or `errored`) and also their status (`Released` or `Not released`).
+
+This help page describes the two possible release management flows on Bump.sh. You can choose between _[Automatic Release](#automatic-release)_ (default for every new documentation) or _[Manual Release](#manual-release)_.
+
+## Automatic releases
+
+:::info
+This is the default release mode for every newly created documentation
+:::
+
+Automatic release mode will publish any new deployed version to your documentation without the need for manual intervention. Everytime you send a new definition file to your documentation, changes will be made to both:
+
+- Your main documentation page. Which will be updated with the latest deployed version.
+- Your documentation changelog page. Which will be updated with the newly released API change.
+
+## Manual releases
```

Comment:
Should we add a `Beta` mention here?

Reply:
It's mentioned in a `caution` callout already, do you want it in the title too?

Reply:
I know I just felt like it would be good to avoid any cognitive burden 😄 but maybe not necessary, leave it up to you or @jayk-u

Reply:
The caution callout looks enough to me.